.github/actions/check-docsite-class-hashes/index.js

```diff
@@ -0,0 +1,53 @@
+import fs from "fs/promises";
+import * as core from "@actions/core";
+
+const classHashesFilePath = "docs/modules/ROOT/pages/utils/_class_hashes.adoc";
+
+async function main() {
+  const compilerVersionInput = core.getInput("compiler-version");
+  const jsonFileInput = core.getInput("json-file");
+
+  let content = await fs.readFile(jsonFileInput, 'utf8');
+
+  // Remove the first 4 lines
+  // TODO: Fix the output of the class hashes to not include the first 4 lines
+  let json = content.split('\n').slice(4).join('\n');
+  let hashes = JSON.parse(json);
+
+  let expectedContent = createHashesFileNewContent(hashes.contracts, compilerVersionInput);
+  let foundContent = await fs.readFile(classHashesFilePath, 'utf8');
+
+  if (expectedContent !== foundContent) {
+    core.debug(expectedContent);
+    core.debug(foundContent);
+    core.setFailed("Class hashes file is not up to date");
```

Comment:
Correct me if I'm wrong, but we need to account for the padding that we've been adding to the hashes. Should we normalize in `starknet_class_hash`?